### PR TITLE
#167923724 Make UI more interactive

### PR DESCRIPTION
**Pull request by EugeneMunya**
#### What does this PR do?
Change the admin dashboard and modals for making UI more interactive
#### Description of Task to be completed?
we corrected dashboard interactivity and we diminished the display modal size 
#### How should this be manually tested?
visiting this link https://eugenemunya.github.io/FreeMentors/UI/
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167923724
#### Screenshots (if appropriate)
Screenshot from 2019-09-10 00-32-31
Screenshot from 2019-09-10 00-31-40

#### Questions:

### CODE REVIEW

**Comment by oesukam** [comment]
@EugeneMunya Check the resource I shared to abide to the correct PR's title

**Comment by EugeneMunya** [comment]
> @EugeneMunya Check the resource I shared to abide to the correct PR's title
Thanks a lot let me correct it

File: UI/styles/modalCss/editSession.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    flex-direction: column;
+    height: 300px;
+    position: relative;
+}
+.modify-session .modal-title{
+    position: absolute;
+    top: 10px;
+    font-size: 16px;
+    font-weight: bold;
+}
```
Comment (oesukam) [inline UI/styles/modalCss/editSession.css]: Blank line required

File: UI/styles/modalCss/mentorReview.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.modal-bg .mentor-review{
+    height: 200px;
+    padding: 1rem;
+}
+.mentor-review .modal-title{
+text-align: center;
+font-size: 16px;
+font-weight: bold;
+padding: 1.5rem;
+}
```
Comment (oesukam) [inline UI/styles/modalCss/mentorReview.css]: Blank line required

File: UI/styles/modalCss/specificMentee.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    display: flex;
+    flex-direction: column;
+}
+
+.modal-bg .modal-title{
+    text-align: center;
+    font-size: 15px;
+    font-weight: bold;
+    padding: 1.2rem;
+    }
```
Comment (oesukam) [inline UI/styles/modalCss/specificMentee.css]: Blank line required

**Pull request merged**